templates-java/src/main/java/br/com/templates_java/config/jms_aws_sqs/SQSDependencyGenerator.java

```diff
@@ -42,4 +42,5 @@ public File runGenerate() throws IOException {
     public void output(String pathPackage, String filename) {
         System.out.println("Add dependencies in ".concat(pathPackage.concat(filename)));
     }
```

Comment:
Variable `pathPackage` could be more descriptive, consider renaming to `packagePath`.